share/spice/geometry/geometry.js

```diff
@@ -56,8 +56,10 @@
             });
             if(svgNode.is(".fill"))
                 svgNode.css("fill", "transparent");
-            if(svgNode.is(".stroke"))
-                svgNode.css("stroke", "#000");
+            if(svgNode.is(".stroke")) {
```

Comment:
Hmm just tested this out, even if I remove the inline changes, and control only via CSS, the color doesn't update when changing theme, any ideas why?